data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/DirectActionsWML/kill_fires_events.cfg

```diff
@@ -0,0 +1,43 @@
+#####
+# API(s) being tested: [kill]fire_event=yes
+##
+# Actions:
+# Create a last breath and die events. Put Alice to the recall list. Kill Bob and Alice, and detect with a variable which events have been triggered for whom.
+##
+# Expected end state:
+# A last breath event and a die event is triggered for Bob but not for Alice.

```

Comment:
"are triggered" instead of "is triggered". Trivial, but might as well change it when cleaning the whitespace.